public/pages/DetectorResults/containers/AnomalyResultsTable.tsx

```diff
@@ -62,6 +67,99 @@ export function AnomalyResultsTable(props: AnomalyResultsTableProps) {
       ? props.anomalies.filter((anomaly) => anomaly.anomalyGrade > 0)
       : [];
 
+  const handleOpenDiscover = async (startTime: number, endTime: number, item: any) => {
+    try {
+      // calculate time range with 10-minute buffer on each side per customer request
+      const TEN_MINUTES_IN_MS = 10 * 60 * 1000;
```

Comment:
nit: would suggest to export a const like this in a common folder. ex https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/telemetry/common/constants.ts#L53